Week5/week5-tasks/src/CalculateTheSumAndAverageFromFile.java

```diff
@@ -0,0 +1,81 @@
+import java.io.*;
+import java.net.URL;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class CalculateTheSumAndAverageFromFile {
+    // Create Private instance from Scanner Class
+    private static final Scanner in = new Scanner(System.in);
+    private static final String WEB_URL = "http://liveexample.pearsoncmg.com/data/Scores.txt";
+    private static final String FILE_URL = "/home/karim/Desktop/internship/JobHacker-System/Week5/week5-tasks/src/Scores.txt";
+    public static void main(String[] args) {
+        // Call for first time only
+        File file = readFromWebToFile(WEB_URL,FILE_URL);
+        File file1 = new File(FILE_URL);
+        System.out.println(getSum(file1));
+        System.out.println(getLength(file1));
+        System.out.println(getAverage(file1));
+    }
+
+    // Function to read from web and save it in file
+    public static File readFromWebToFile(String url, String fileUrl){
```

Comment:
make the method only take one input as string the url and return a file

Reply:
Done